android/src/main/kotlin/com/authgear/flutter/AuthgearPlugin.kt

```diff
@@ -388,6 +440,15 @@ class AuthgearPlugin: FlutterPlugin, ActivityAware, MethodCallHandler, PluginReg
       sharedPreferences.edit().putString(key, value).commit()
       result.success(null)
     } catch (e: Exception) {
+      // NOTE(backup): Please search NOTE(backup) to understand what is going on here.
+      if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.KITKAT) {
+        if (e is AEADBadTagException) {
```

Comment:
Suggest to catch `GeneralSecurityException` here too.

Reply:
Fixed.